include/dlaf/eigensolver/tridiag_solver/index_manipulation.h

```diff
@@ -147,7 +147,7 @@ inline void invertIndex(const SizeType i_begin, const SizeType i_end,
   namespace di = dlaf::internal;
 
   const SizeType n = problemSize(i_begin, i_end, in.distribution());
-  auto inv_fn = [n](const auto& in_tiles_futs, const auto& out_tiles, [[maybe_unused]] auto&&... ts) {
+  auto inv_fn = [n](const auto& in_tiles_futs, const auto& out_tiles) {
```

Comment:
Just checking: do you remember if the pack was there because of the old implementation instantiating this function also with a stream or something in that direction? And that is no longer needed?

Reply:
Originally it was implemented for both CPU and GPU, then we dropped the GPU implementation and we had this leftover from the "kernel cleanup".